src/navigation/AppStackNavigator.js

```diff
@@ -21,6 +23,18 @@ const AppStackNavigator = (headerRight = true) =>
           title: props.navigation.getParam('title', texts.screenTitles.home)
         })
       },
+      ConstructionSiteDetail: {
+        screen: ConstructionSiteDetailScreen,
+        navigationOptions: () => ({
+          title: texts.screenTitles.constructionSite
```

Comment:
detail title should be singular, like in other detail screens

Reply:
done in cd9815e1e0e6c8923c384d1382e3843d21f2bad3